### ISSUE TITLE
build probelm with libnorm src/udp_engine.cpp:33:27: error: 'close' was not declared in this scope

### ISSUE THREAD

**Issue by UsamaMehboob** [zeromq/libzmq#1754]
There is a build problem, though minute, when built libzmq with libnorm. Here are the steps to reproduce the problem. I am using ubuntu 15.10 (64)
1. built the libnorm.so
2. git clone https://github.com/libzmq.git
3. sh autogen.sh
4. ./configure --with-norm=/path/to/norm/tree
5. make
After step 5 this error produced
```
src/udp_engine.cpp: In destructor 'virtual zmq::udp_engine_t::~udp_engine_t()':
src/udp_engine.cpp:33:27: error: 'close' was not declared in this scope
         int rc = close (fd);
```
After a little googling, I added a line `#include <unistd.h>` in udp_engine.cpp. This error got fixed then. But then another error arose `cannot find -lnorm`; which was fixed by placing libnorm.so in /usr/local/lib.

**Comment by UsamaMehboob** [comment]
I have submitted a pull request for this.

**Comment by c-rack** [comment]
And it's merged: #1755, thanks!
